src/Dropdown/index.js

```diff
@@ -321,5 +328,5 @@ Dropdown.defaultProps = {
 export default styled(Dropdown)`
   position: relative;
   user-select: none;
-  height: 100%;
+  height: ${({ height }) => height || '3.8rem'};
```

Comment:
Had to fallback to the default prop again since it is undefined, it does not seem that the styled component retrieve the default prop value